packages/typespec-test/test/healthInsights_radiologyinsights/generated/typespec-ts/package.json

```diff
@@ -11,8 +11,10 @@
     "exports": {
       "./package.json": "./package.json",
       ".": "./src/index.ts",
+      "./api": "./src/api/index.ts",
```

Comment:
Is this expected? This is starting with @azure-rest with modular client.

Reply:
issue created https://github.com/Azure/autorest.typescript/issues/2999